docs/source/model_design/vivarium_model_components/impairments/index.rst

```diff
@@ -33,17 +33,224 @@ Modeling Impairments
+
+For the case of blindness, there are approximately 15 separate causes, not including 
+all neonatal causes. This would mean having 15 cause models included! This is 
+likely to be a lot of work, and would probably not provide much extra information. 
+
+Furthermore, you still have the issue that cause models are independent in Vivarium. 
+Simulants could still get blindness due to multiple causes, leading to the same undercounting 
+of total blindness. 
+
+For practical and logistic reasons, this approach is not recommeneded. 
+
+Include a Hybrid Cause & Impairment Model
++++++++++++++++++++++++++++++++++++++++++
+
+Another option is to include a single cause model that includes both the cause 
+and impairment. One example of this is in the CVD model where IHD and Heart Failure 
+are included in the same :ref:`cause model <2019_cause_ihd>`.
+
+By including both the cause and impairment in a single modeling component, 
+they are no longer independent which solves our prior problems of overlap. 
+In addition, you can define part of the model to overlap - for example states 
+could inlcude diabetes, blindness due to diabetes, and blindness due to other 
+causes. 
+
+Including cause models structured in this way does sometimes lead to strange "predictive" 
+qualities for simulants. For example, a simulant with blindness from other 
+causes might not be able to develop diabetes to avoid overlap with blindess 
+due to diabetes. In real life, these conditions exist separately, but can't in 
+the model. 
+
+This approach should not create bias in your model and generally works well. 
+However, if you want to include many causes that overlap with an impairment 
+(e.g., including diabetes, meningitis, glaucoma, and vitamin A deficiency, all of 
+which cause blindness) 
+using this approach would make an extremely complex single "cause" model that 
+likely wouldn't be feasible to create or V&V. 
+
+Therefore, considering your modeling situation and the requirements is important 
+in choosing your modeling approach. 
+
+Have Causes Act as "Risks"
+++++++++++++++++++++++++++
+
+Another option is to have cause models that act as "risks" for other 
+causes. To understand this, imagine a model with both diabetes and 
+the blindness impairment included. As we discussed 
+above, causes in Vivarium are usually independent, but we can have causes 
+act on other transition pathways. 
+
+For example, if a simulant is in the state within the diabetes cause 
+for "blindness due to diabetes", it could prevent them from transitioning 
+from "susceptible" to "blindness due to other causes" in the blindness impairment. 
+Hence, a cause (diabetes) acts on a transition in the impairment component. 
+
+You could imagine a similar pathway from the blindness impairment to 
+the diabetes cause model. This essentially enforces the idea that a simulant 
+can only go blind by 1 cause. This could in theory scale up for 
+an increasing number of cause and/or impairment models. 
```

Comment:
I'm not sure we should recommend this way. It seems like in order to avoid bias you would need to adjust the blindness impairment incidence for the change in denominator, and I'm not sure it is any easier than the next approach discussed.